src/controllers/userController.js

```diff
@@ -7,6 +7,7 @@ import messages from '../utils/messages';
 import {
   create, findByEmail, comparePasswords, findByEmailOrPhone
 } from '../services/userServices';
+// import notification from "../services/notificationServices";
```

Comment:
no comments

Reply:
I don't know how that got there.. but its gone now 😊